StabilityPackage.wl

```diff
@@ -603,5 +607,53 @@ Return[out];
 
 
 
+(* Caclulates the 3 angular moments from a DO file and returns a list of the 3 moments from radial index rsrt to radial index rend. *)
+getDOmoments[dofile_,rsrt_,rend_]:=Module[{dat,srdat,out,emom,fmom,pmom},
+dat=ImportData[dofile];
+out=Reap[
+	Do[
+srdat=SelectSingleRadius[dat,ri];
+emom={Sum[Sum[Sum[srdat["Endensity"][[1,f,dt,dp]] / (h (srdat["freqmid"][[f]])),{f,1,Length[dat["freqs"]]-1}],{dp,1,Length[dat["phis"]]-1}],{dt,1,Length[srdat["mids"]]}],

```

Comment:
Why are we summing over all species? Don't we want a separate moment for each?

Reply:
This isn't summed over all species, it's a list {species 1, species 2, species 3} where each entry has sums in it. i.e. species1=sum[sum[sum[stuff]]]]. That's why the first index (the species index) is hard coded and not a summation variable.

Reply:
Oh, I got confused by the github "+" indicating that a line was added. Thanks!